ui/pages/Login/components/AcceptPolicies.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Link } from 'react-router-dom'
+
+import { BooleanCheckbox } from 'shared/components/form/Inputs'
+import { validators } from 'shared/components/form/ReduxFormWrapper'
+
+import { updateUserPolicies } from '../reducers'
+import UserFormLayout from './UserFormLayout'
+
+
+const POLICY_FORM_FIELDS = [
+  {
+    name: 'acceptedPolicies',
+    component: BooleanCheckbox,
+    validate: validators.required,
+    label: (
+      // eslint-disable-next-line jsx-a11y/label-has-for
+      <label>
+        I accept the <Link target="_blank" to="/terms_of_service">Terms of Service</Link> and
+        the <Link target="_blank" to="/privacy_policy">Privacy Policy</Link>
+      </label>
+    ),
+  },
+]
+
+const AcceptPolicies = React.memo(({ onSubmit }) =>
+  <UserFormLayout
+    header="Seqr Policies"
+    subheader="Before continuing to use seqr, please read and accept our policies"
+    onSubmit={onSubmit}
+    form="acceptPolicies"
+    fields={POLICY_FORM_FIELDS}
+  />,
```

Comment:
The `UserFormLayout` component has a header bar with some buttons on it. It seems that it is better to disable these buttons except `loggout`.

Reply:
so actually, this header has no concept of which page it is on, so changing this is incredibly difficult. If a user clicks any of those links and tries to view any data it will just redirect them back to the accept policies page, so I don't think leaving this as-is will cause any harm

Reply:
Okay